src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Compute/Add-ArmMonitoringExtension.ps1

```diff
@@ -0,0 +1,38 @@
+function Add-ArmMonitoringExtension {
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    Param(
+        [PSTypeName("VirtualMachineScaleSet")]
+        [Parameter(Mandatory, ValueFromPipeline)]
+        $VirtualMachineScaleSet,
+        [PSTypeName("ServiceFabricNodeType")]
+        [Parameter(Mandatory)]
+        $NodeType,
```

Comment:
This extension is not specific to service fabric. We should change the parameter to take in a Name